mock/docs/buildroot-lock-schema-1.0.0.json

```diff
@@ -82,8 +82,8 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "image_digest": {
-          "description": "Digest got by the 'podman image inspect --format {{ .Digest }}' command, sha256 string",
+        "image_layers_digest": {
+          "description": "SHA256 digest concatenated layer digests from 'podman image inspect --format {{ .RootFS }}' command, sha256 string",
```

Comment:
Do we want to keep both?  I'm not a jsonschema guru, does this imply a schema version change @xsuchy ?

Reply:
I'm not a JSON person :-) this was my first scheme I prepared... 

But I think I'd prefer to add version 2.0.0 here, or 1.1 (if we keep the old `image_digest` field).  Or rather radical change - drop the versioning entirely (because it seems just wrong to change the schema within the same version this way).

Reply:
Still relevant;  + we should document that we are using the `.Config` value, too